orchestra/utils/task_lifecycle.py

```diff
@@ -1031,6 +1031,19 @@ def update_related_assignment_status(task, assignment_counter, data,
         start_datetime=submit_datetime)
 
 
+def _call_abort_completion_function(project):
+    abort_completion_fn = project.workflow_version.abort_completion_function
+    abort_completion_fn_path = abort_completion_fn.get('path', '')
+    abort_completion_fn_kwargs = abort_completion_fn.get('kwargs', {})
+    if len(abort_completion_fn_path) == 0:
```

Comment:
It's not an error if we don't have one, so let's not warn. Instead, let's put lines 1041 and 1042 in an `if abort_completion_fn_path:`